docs/advanced-topics/docker-alternatives.rst

```diff
@@ -73,6 +73,36 @@ add a line to your ``~/.dockstore/config``:
 
     cromwell-vm-options: -Dconfig.file=<absolute path to your Cromwell conf>
 
+Podman
+---------------
+
+Podman is a daemon-less alternative of Docker that allows end users to run docker/Open Container Initiative (OCI) containers without root privileges.
+
+Podman can be installed following the instructions
+`here <https://podman.io/getting-started/installation>`__.
+
+cwltool
+~~~~~~~
+
+Podman is also available as a command line option for cwltool like this:
+::
+
+    cwltool --podman <workflow> <input json>
+
+To set this option through Dockstore, add the following line to your ``~/.dockstore/config``:
```

Comment:
Reword to "to set this as the default option when running via the Dockstore CLI" or something else that makes it clear this is for the Dockstore CLI, not Dockstore as a product/website/service (yes, this has confused people before).